kolibri/deployment/default/settings/base.py

```diff
@@ -195,5 +195,10 @@
 
 AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
 
+AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"
```

Comment:
Yeah, makes sense: settings defaults should go in the app (that's where the setting is "defined"), and this file is just for overriding settings at the project-level.

Please advise on how best to handle the following. I need to have DRF's settings depend on the `AUTH_ANONYMOUS_USER_MODEL` setting:

```python
REST_FRAMEWORK = {
    "UNAUTHENTICATED_USER": AUTH_ANONYMOUS_USER_MODEL
}
```

(This is needed because DRF bypasses the main Django authentication middleware, and does things itself, creating anon users based on the `REST_FRAMEWORK["UNAUTHENTICATED_USER"]` setting.)

How/where would one cleanly set this dependency? Note that it doesn't need to be dynamically configured in practice, so one option is just to not have a dependency; just set `REST_FRAMEWORK = {"UNAUTHENTICATED_USER": "kolibri.auth.models.KolibriAnonymousUser"}` in deployment settings, overriding the DRF app default, and then set `AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"` separately in the auth app settings. This is less DRY, perhaps, but doesn't create unnecessary dependencies, so might be best?